### PR TITLE
Converting research data to molecule 3

### LINKED ISSUE #1966: convert researchdata to molecule 3

### PR DESCRIPTION
**Pull request by carolyncole**
fixes #1966

### CODE REVIEW

File: roles/researchdata/molecule/default/verify.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    check_mode: true
+    register: present
+    failed_when:
+      - present is changed
+  - name: test for postgres database and login
+    command: psql -U researchdata -w researchdata -c '\d'
+    register: postgres_results
+    ignore_errors: true
+    changed_when: false
+  - name: assert cron jobs exists
```
Comment (kayiwa) [inline roles/researchdata/molecule/default/verify.yml:28]: better label maybe? we are asserting a DB exists.

**Review by kayiwa — COMMENTED** [review]
a better labeled assert maybe?

File: roles/researchdata/molecule/default/verify.yml (hunk 1 above)
Comment (carolyncole) [reply roles/researchdata/molecule/default/verify.yml]: Updated...

**Review by kayiwa — APPROVED** [review]
OK

**Pull request merged**